samples/hybrid-connections/java/simple-http-demo/src/main/java/HttpSender.java

```diff
@@ -0,0 +1,75 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.time.Duration;
+import java.util.Scanner;
+import com.microsoft.azure.relay.RelayConnectionStringBuilder;
+import com.microsoft.azure.relay.TokenProvider;
+
+public class HttpSender {
+	static final String CONNECTION_STRING_ENV_VARIABLE_NAME = "RELAY_CONNECTION_STRING";
+	static final RelayConnectionStringBuilder connectionParams = new RelayConnectionStringBuilder(
+			System.getenv(CONNECTION_STRING_ENV_VARIABLE_NAME));
+
+	public static void main(String[] args) throws IOException {
+		TokenProvider tokenProvider = TokenProvider.createSharedAccessSignatureTokenProvider(
+				connectionParams.getSharedAccessKeyName(), 
+				connectionParams.getSharedAccessKey());
+
+		// For HTTP connections, the scheme must be https://
+		StringBuilder urlBuilder = new StringBuilder(connectionParams.getEndpoint().toString() + connectionParams.getEntityPath());
+		urlBuilder.replace(0, 5, "https://");
```

Comment:
>urlBuilder.replace(0, 5, "https://"); [](start = 2, length = 37)

I'm not saying fix it now, but it kind of bothers me that this assumes the endpoint starts with "sb://" but doesn't verify that's the case.